keen/saved_queries.py

```diff
@@ -57,15 +60,18 @@ def results(self, query_name):
 
     def create(self, query_name, saved_query):
         """
-        Creates the saved query via a PUT request to Keen IO Saved Query endpoint. Master key must be set.
+        Creates the saved query via a PUT request to Keen IO Saved Query endpoint.
+        Master key must be set.
         """
         keen_api = KeenApi(self.project_id, master_key=self.master_key)
         self._check_for_master_key()
         url = "{0}/{1}/projects/{2}/queries/saved/{3}".format(
             keen_api.base_url, keen_api.api_version, self.project_id, query_name
         )
+
+        payload = json.dumps(saved_query)
```

Comment:
I vote we do some type checking here, to see if it is a string or dict. That also makes this a non-breaking change.